src/lib/permissions/rules.ts

```diff
@@ -1,6 +1,58 @@
 import { rule } from 'graphql-shield';
+import { Context } from '../../context';
 
-export const isAuthenticated = rule({ cache: 'contextual' })(async (_, __, ctx) => {
-    console.log(ctx.userId);
+export const isAuthenticated = rule({ cache: 'contextual' })(async (_, __, ctx: Context) => {
     return ctx.userId !== null;
 });
+
+export const isParticipantOfMeeting = rule({ cache: 'contextual' })(async (_, { meetingId }, ctx: Context) => {
+    const participant = ctx.prisma.participant.findFirst({
+        where: {
+            userId: ctx.userId,
+            meetingId,
+        },
+    });
+    return participant !== null;
+});
+
+export const isParticipantOfVotation = rule({ cache: 'contextual' })(async (_, { votationId }, ctx: Context) => {
+    const votationWithParticipant = await ctx.prisma.votation.findFirst({
```

Comment:
could use count instead of findfirst

Reply:
Can also 

```
await ctx.prisma.votation.count({
        where: { meeting: { participants: { some: { userId: ctx.userId } } } },
    });
```

Reply:
I would probably also use the Role to check if hes a participant

Reply:
I realise the double meaning of participant may be confusing, but the idea is that participant in this context refer to owners and counters as well as regular participants. Maybe we should consider renaming the role "participant" to something else in order to avoid this confusion?

Reply:
I was looking for something like the some-command you suggested there without finding it, that is definitely better then how it is now.